docs/workflows.md

```diff
@@ -1 +1,94 @@
-TODO
\ No newline at end of file
+# Workflows
+
+The purpose of this section of the Kn documentation is to list what we believe are common workflows or use-cases for the Knative CLI. Obviously, as we release the CLI in the wild and get users in production, we might discover novel workflows. As we do we will improve this document.
+
+## Basic (get your feet wet)
+
+In this basic worflow we show the CRUD (create, read, update, delete) operations on a service. We use a well known [simple Hello World service](https://github.com/knative/docs/tree/master/docs/serving/samples/hello-world/helloworld-go) that reads the environment variable `TARGET` and prints it as output.
+
+1. Create a service from image in `default` namespace
```

Comment:
Your markdown renders the numbered list as 1. 1. 1, rather than numbering properly.

Reply:
Yeah, was afraid of that. I'll use bullet points. Thx.